### ISSUE TITLE
PaymentRequest IPC connection error

### ISSUE THREAD

**Issue by willhowlett** [Adyen/adyen-web#2672]
**Describe the bug**
We use adyen-web 5.55.1

Since around 5pm on Monday 22nd April we’ve had the following error on our payments page on a subset of Android browsers

`Renderer process could not establish or lost IPC connection to the PaymentRequest service in the browser process`

This seems to only be affecting the following or very similar, i.e. `SamsungBrowser`) useragent

`Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/24.0 Chrome/117.0.0.0 Mobile Safari/537.36`

I’m not aware of the [PaymentRequest API](https://gbr01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FAPI%2FPayment_Request_API&data=05%7C02%7Cwillhowlett%40hollandandbarrett.com%7C858eeb37c930464ac0dc08dc6b3a817f%7Cf8a60c52c7ce4389b0ff3c91bacf6d24%7C0%7C0%7C638503146233446535%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=X2J0HxDAXyj%2FbMPpyp%2Bw%2BGbx6xJfssj8mElHtPtaBFM%3D&reserved=0) being used in adyen-web (have searched the codebase), but is this in use by your components (v5) in some form and do you know what might be causing this error?

I have raised an Adyen support ticket but they suggested posting here

Many thanks

**Comment by longyulongyu** [comment]
Hi @willhowlett thanks for reporting.

In adyen-web, we are not using the PaymentRequest api, but it could be that some express payments such as Google Pay or Apple Pay uses it, and [not all browsers support it](https://developer.mozilla.org/en-US/docs/Web/API/Payment_Request_API#browser_compatibility).

If you support the express payments, could you disable them for the mentioned user agent and check if the issue persists?

If you don't support the express payments, we'd like to know how you integrate with adyen-web sdk, are you using WebView or something similar? Which payment method(s) caused the issue or the whole adyen-web was not rendering?

**Comment by sponglord** [comment]
@willhowlett - any update on this issue? Did you try the above advise?

**Comment by willhowlett** [comment]
Apologies, I overlooked this

The errors are happening on our main checkout page where we do not have any express payments enabled

We are on `5.55.1` of adyen-web, rendered directly in our app (no webview or anything)

As far as I can tell it is not stopping any payment methods from rendering. I was able to see from our logs that I had unknowingly had the error happen during my session when testing for the issue in browserstack, but there was no errors for me in the frontend when this happened. Based on that it seems like there is no direct customer impact, it would just be good to understand why the error is happening if possible.

Many thanks
